src/lib/roc/config.h

```diff
@@ -105,6 +105,9 @@ typedef struct roc_receiver_config {
 
     //! A bitmask of ROC_FLAG_* constants.
     unsigned int flags;
+
+    //! Number of samples for all channels per second.
+    unsigned int sample_rate;
```

Comment:
This comment looks ambiguous. I suggest "Number of samples per second per channel". We should also fix corresponding comments in ReceiverConfig and SenderConfig.

Reply:
Fixed